### PR TITLE
Add a draft for a blog entry

### PR DESCRIPTION
**Pull request by gassmoeller**
This is a draft for the promised blog entry about my BSSW fellowship @bernhold. I have one question concerning figures: Should I make a  separate PR to betterscientificsoftware/images for them? And is there a way to preview the figures in the final text? I would like to see if they have an appropriate size and how they fit into the sections.

Let me know what you think about it, I am happy to modify it.

### CODE REVIEW

**Comment by hnamLANL** [comment]
I updated with some small fixes and a note that can be pursued or ignored.

**Comment by rinkug** [comment]
I have reviewed this and it looks good overall. Fixed a few more typos. I saw a note from Gail in there for clarification on a particular sentence (IMO, the sentence makes sense as is and Gail's note can be  disregarded but I leave it up to David/Rene to decide)

File: Articles/Blog/2020-03-software-communities.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+relatively little about the challenges that are specific to scientific
+software.
+
+### Community engagement as a limit to project growth?
+
+I am a scientist who develops software, and I regularly meet with others in my field (geophysics) to improve their software development
+processes. During my work, I have frequently noticed that some software project members have
+valuable scientific ideas and follow best practices for software design yet
+never attract a sufficient user base to establish themselves. 
+(%GAIL asks- not sure wheat you mean by "establish themselves" - I guess "as a thriving community" but that seems repetitious, and establish themselves as scientists is not what you mean, I am sure)
```
Comment (gassmoeller) [inline Articles/Blog/2020-03-software-communities.md]: I mean "establish themselves" as in "establish themselves as ongoing projects with thriving communities." which would require rephrasing the next sentence. I will leave the decision up to @bernhold if this needs clarification here, I am fine modifying it.

**Comment by gassmoeller** [comment]
Thank you all for the comments, I think the text is much better now. I am fine with the current version, just waiting for some guidance on the one remaining question.

File: Articles/Blog/2020-03-software-communities.md (hunk 1 above)
Comment (bernhold) [reply Articles/Blog/2020-03-software-communities.md]: I added "their software project" to clarify without redundancy

**Pull request merged**